src/main/resources/resources/ManagerResources_en.properties

```diff
@@ -1748,4 +1748,5 @@ info.manage.associated.objects.no.administrativeOffices=There are no administrat
 info.manage.associated.objects.no.emptyDegree=There is no empty degree.
 title.manage.associated.objects.departments.edit=Edit Department
 title.manage.associated.objects.administrativeOffices.create=Create Administrative Office
-info.manage.associated.objects.emptyDegree.associated.with=Empty degree associated with:
\ No newline at end of file
+info.manage.associated.objects.emptyDegree.associated.with=Empty degree associated with:
+warning.manager.SIBS.outdate.code=The payment code {0} ({1}) is old and should already be registered. Check this manually.
```

Comment:
@sergiofbsilva There is a typo in the key here. It should be "warning.manager.SIBS.outdated.code" 🔍